src/fidesops/ops/schemas/manual_webhook_schemas.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Optional, Set
+
+from fideslib.utils.text import to_snake_case
+from pydantic import ConstrainedStr, validator
+
+from fidesops.ops.schemas.base_class import BaseSchema
+from fidesops.ops.schemas.connection_configuration.connection_config import (
+    ConnectionConfigurationResponse,
+)
+
+
+class ManualWebhookFieldType(ConstrainedStr):
+    """Using ConstrainedStr instead of constr to keep mypy happy"""
+
+    min_length = 1
+    max_length = 200
+
+
+class ManualWebhookField(BaseSchema):
+    """Schema to describe the attributes on a manual webhook field"""
+
+    pii_field: ManualWebhookFieldType
+    dsr_package_label: Optional[ManualWebhookFieldType]
```

Comment:
Should this default to `None`?

```suggestion
    dsr_package_label: Optional[ManualWebhookFieldType] = None
```